docker/host.sls

```diff
@@ -53,5 +53,13 @@ docker-py:
     - reload_modules: True
 {% endif %}
 
+{%- if host.registry is defined %}
+
+docker_login:
+  cmd.run:
```

Comment:
You should use salt module instead of cmd.run:
https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.dockerio.html#salt.modules.dockerio.login